avm/res/app/container-app/main.bicep

```diff
@@ -217,6 +220,7 @@ resource containerApp 'Microsoft.App/containerApps@2024-03-01' = {
       ingress: disableIngress
         ? null
         : {
+            additionalPortMappings: !empty(additionalPortMappings) ? additionalPortMappings : null
```

Comment:
```suggestion
            additionalPortMappings: additionalPortMappings
```
As it is nullable, the default would anyways be `null`

Reply:
Ah, good point.